src/otx/core/model/entity/classification.py

```diff
@@ -77,11 +77,11 @@ def forward_explain(
         self.model.explain_fn = self.get_explain_fn()
 
         # If customize_inputs is overridden
-        outputs = (
-            self._forward_explain_image_classifier(self.model, **self._customize_inputs(inputs))
-            if self._customize_inputs != ExplainableOTXClsModel._customize_inputs
-            else self.model(inputs)
-        )
+        if self._customize_inputs != ExplainableOTXClsModel._customize_inputs:
+            customized_inputs = self._customize_inputs(inputs)
+        else:
+            customized_inputs = {inputs}
```

Comment:
Should a set of inputs here be created here?
Double asterisks operator would work only with dict, with set (not mapping, just iterable object) you would need just one asterisk.

Do we have a case when `self._customize_inputs == ExplainableOTXClsModel._customize_inputs` to test it? I do not see one..

Reply:
Me too, actually. I'm not sure in which case this happens. 
I put inputs in dict in `else` branch exactly for the purpose of getting initial `inputs` after passing that as `**{inputs}` below. The type of `inputs` shouldn't affect that

Reply:
with `obj = {value}`, obj is a set, not a dict

Reply:
Thank you, I missed it